frontend/App.js

```diff
@@ -6,22 +6,25 @@ import { createStackNavigator } from '@react-navigation/stack';
 import LoginScreen from './components/LoginScreen.js';
 import SplashScreen from './components/SplashScreen.js';
 import DashboardScreen from "./components/DashboardScreen";
+import EducationScreen from './components/EducationScreen.js';
 import Navbar from './components/Navbar.js';
 
+
 const Stack = createStackNavigator();
 
 export default function App() {
   return (
     <NavigationContainer>
-      <Stack.Navigator
-        initialRouteName="Splash"
-        screenOptions={{
-          headerShown: false,
-        }}
+      <Stack.Navigator 
+      initialRouteName="Home" 
```

Comment:
You changed the initial route to 'home', but we don't have a Stack.Screen name equaling home, what's the effect on the app?

Reply:
Oh yeah, good call out. I honestly don't know why I did this haha